src/main/kotlin/org/hyacinthbots/lilybot/extensions/moderation/TerminalModeration.kt

```diff
@@ -349,23 +348,22 @@ class TerminalModeration : Extension() {
 					content = "Kicked User"
 				}
 
-				actionLog.createMessage {
-					embed {
-						title = "Kicked a user"
-						description = "${userArg.mention} has been kicked!"
-						image = arguments.image?.url
-						baseModerationEmbed(arguments.reason, userArg, user)
-						dmNotificationStatusEmbedField(arguments.dm, dm)
-						timestamp = Clock.System.now()
-					}
-				}
-
 				if (config.publicLogging != null && config.publicLogging == true) {
 					channel.createEmbed {
 						title = "Kicked a user"
 						description = "${userArg.mention} has been kicked!"
 					}
 				}
+
+				if (!configIsUsable(ConfigOptions.ACTION_LOG, guild!!.id)) return@action
+				actionLog.createEmbed {
```

Comment:
General standard we've been running across the board is ``channel.createMessage { embed { //... } }``. Please switch to that on applicable places